app/views/bookings/edit.html.erb

```diff
@@ -1,6 +1,19 @@
+<div class="container">
+  <div class="row">
+    <div class="col-12 col-md-6 offset-md-3">
+
 <%= simple_form_for @booking do |f| %>
+ <div class="row justify-content-center"></div>
```

Comment:
We should make sure that the styles we use in our forms are all the same. @jselvaraj95 could you and @brovani make sure that you aren't both working on the same thing?